src/lib/i18n/LanguageKeys/Commands/Reddit.ts

```diff
@@ -5,14 +5,21 @@ export const RootName = T('commands/reddit:name');
 export const RootDescription = T('commands/reddit:description');
 
 export const OptionsReddit = 'commands/reddit:optionsReddit';
+export const OptionsPost = 'commands/reddit:optionsPost';
+
+export const Subreddit = 'commands/reddit:subreddit';
+
+export const PostCommand = 'commands/reddit:post';
```

Comment:
Rename constant to `Post` and put it together `Subreddit`.

Reply:
There is already another constant named Post

Reply:
Rename the other constant to `PostResult`, in that case.